cli/src/config.rs

```diff
@@ -349,13 +369,13 @@ impl ConfigEnv {
 
     /// Returns a path to the repo-specific config file.
     pub fn repo_config_path(&self) -> Option<&Path> {
-        self.repo_config_path.as_ref().map(ConfigPath::as_path)
+        self.repo_config_path.as_ref().map(|p| p.as_path())
     }
 
     /// Returns a path to the existing repo-specific config file.
     fn existing_repo_config_path(&self) -> Option<&Path> {
-        match &self.repo_config_path {
-            Some(ConfigPath::Existing(path)) => Some(path),
+        match self.repo_config_path {
```

Comment:
nit: `&self.repo_config_path` instead of `ref path` as before? (I don't have strong preference about that, though.)

Reply:
I don't really want to change this.

Reply:
Fair. I just pointed out because you've changed the existing style.